template.json

```diff
@@ -1,281 +1,351 @@
-            "type": "Number",
+          "pil" : {
+            "type" : "Number",
             "label" : "SKILL.Pil",
             "attribute" : "a",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "psy": {
-            "type": "Number",
+          "psy" : {
+            "type" : "Number",
             "label" : "SKILL.Psy",
             "attribute" : "wil",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "sch": {
-            "type": "Number",
+          "sch" : {
+            "type" : "Number",
             "label" : "SKILL.Sch",
             "attribute" : "int",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "sth": {
-            "type": "Number",
+          "sth" : {
+            "type" : "Number",
             "label" : "SKILL.Sth",
             "attribute" : "a",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "surv": {
-            "type": "Number",
+          "surv" : {
+            "type" : "Number",
             "label" : "SKILL.Surv",
             "attribute" : "wil",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "tech": {
-            "type": "Number",
+          "tech" : {
+            "type" : "Number",
             "label" : "SKILL.Tech",
             "attribute" : "int",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "ws": {
-            "type": "Number",
+          "ws" : {
+            "type" : "Number",
             "label" : "SKILL.WS",
             "attribute" : "i",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           }
         }
       },
-      "survival": {
-        "survival": {
-          "wounds": {
-            "value": 0,
-            "max": 0
+      "combattraits" : {
+        "combattraits" : {
+          "wounds" : {
+            "value" : 0,
+            "bonus" : 0,
+            "max" : 0
           },
-          "shock": {
-            "value": 0,
-            "max": 0,
-            "determination" : 0
+          "shock" : {
+            "value" : 0,
+            "bonus" : 0,
+            "max" : 0
           },
           "defence" : {
             "value" : 0,
-            "base" : 0,
+            "bonus" : 0,
+            "total" : 0
+          },
+          "resilience" : {
+            "value" : 0,
+            "bonus" : 0,
             "armour" : 0,
             "total" : 0
+          },
+          "determination" : {
+            "value" : 0,
+            "bonus" : 0,
+            "total" : 0
           }
         }
       },
-      "objective": {
-        "objective": {
-          "value" : "",
-          "conviction" : 0,
-          "size" : "",
-          "resolve" : 0,
-          "speed" : 0
+      "mentaltraits" : {
```

Comment:
Can this be renamed to "mental"?

Reply:
Definitely, I shoulda used camel case anyways